setup.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+from distutils.cmd import Command
+class TestCommand(Command):
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        import sys, subprocess
+
+        raise SystemExit(
+            subprocess.call(['nosetests',
```

Comment:
sure you can set it many ways. 

   subprocess.Popen('nosetests', cwd='/blah')

I called nosetests like this because it was easy. pyunit was not picking up the tests.